content/en/docs/zero-code/python/operator.md

```diff
@@ -3,11 +3,26 @@ title: Using the OpenTelemetry Operator to Inject Auto-Instrumentation
 linkTitle: Operator
 aliases: [/docs/languages/python/automatic/operator]
 weight: 30
-cSpell:ignore: distro mkdir uninstrumented virtualenv
+cSpell:ignore: distro grpcio mkdir psutil uninstrumented virtualenv
 ---
 
 If you run your Python service in Kubernetes, you can take advantage of the
 [OpenTelemetry Operator](https://github.com/open-telemetry/opentelemetry-operator)
 to inject auto-instrumentation without having to modify each of your services
 directly.
 [See the OpenTelemetry Operator Auto-instrumentation docs for more details.](/docs/kubernetes/operator/automatic/)
+
+### Python specific topics
+
+#### Libraries with binary wheels
+
+Some Python packages we instrument or need in our instrumentation libraries,
+might ship with some binary code. This is the case, for example, of `grpcio` and
+`psutil` (used in `opentelemetry-instrumentation-system-metrics`).
```

Comment:
```suggestion
Some Python packages we instrument or need in our instrumentation libraries
might ship with some binary code. This is the case, for example, with `grpcio` and
`psutil`, which are used in `opentelemetry-instrumentation-system-metrics`.
```

Reply:
Only psutil is used in the system metrics package, grpcio is used generally in the exporters or in its own instrumentation library.